src/apiCalls.js

```diff
@@ -1,13 +1,51 @@
 const apiCalls = {
 
     getGameData: () => {
-        return fetch('https://api.boardgameatlas.com/api/search?name=Catan&pretty=true&client_id=JLBr5npPhV')
-            .then(res => res.json())
+        return fetch('https://api.boardgameatlas.com/api/search?limit=25&pretty=true&client_id=YWBLPm0lyY')
+            .then(res => {
+                if (res.ok) {
+                    return res.json()
+                } else {
+                    throw new Error()
+                }
+            })
+            .catch(err => err)
+    },
+
+    getSpecificGame: (gameId) => {
```

Comment:
So helpful to have these all in this file and be able to call methods on the apiCalls object. Thank you Joan!